docs/scipp-neutron/instrument-view.ipynb

```diff
@@ -62,7 +62,8 @@
    },
    "outputs": [],
    "source": [
-    "sample = sn.load(filename='PG3_4844_event.nxs')"
+    "sample = sn.load(filename='PG3_4871_event.nxs')\n",
```

Comment:
Maybe better to refer to `data/fetch_neutron_data.py` and change back to PG3_4844